docs/graph_analytics_workloads.md

```diff
@@ -1,27 +1,43 @@
 
-In GraphScope, Graph Analytics Engine (GAE) is responsible for handling such graph analytics algorithms, which allows users to easily define and implement their algorithms while providing efficient execution for these algorithms.
+![Applications of graph analytics](./images/analytics_examples.png)
 
-### Rich built-in graph analytics algorithms
 
-GAE of GraphScope provides 20 graph analytics algorithms as built-in algorithms, and users can directly invoke them. The build-in algorithms contain most commonly used algorithms, including PageRank, BFS, DFS, shortest path and LCC. In addition, GraphScope is compatible with NetworkX APIs, and thus diverse kinds of [built-in algorithms in NetworkX](https://networkx.org/documentation/stable/reference/algorithms/index.html) can also be directly invoked by users. In total, over 100 build-in graph analytical algorithms can be directly executed over GraphScope, without any developing effort.
+## Characteristics of Graph Analytics Workloads
+Based on our experience, the graph data we need to process and the frameworks (systems) used to process graph data have the following characteristics:
 
+### Large-scale and complex graph data
 
-### Customize your own algorithms
+We have observed that vast majority of real-world graph data is large-scale, heterogeneous, attributed. For example, nowadays e-commerce graphs often contain billions of vertices and tens of billions of edges, with various types and rich attributes. How to represent and store such graph data is nontrivial.
 
-In many cases, users need to develop their customized graph analytics algorithms. To this end, GraphScope allows users to succinctly develop their own algorithms, with multiple programming models and programming languages support.
+### Various programming models/languages
+
+Currently, many graph processing systems have been developed to handle graph analytics algorithms. These systems are implemented with different programming models (e.g., vertex-centric model and PIE model) and programming languages (e.g., C++, Java and Python). As a result, users usually suffer from high learning curves.
+
+### Requirement for high performance
+
+The scale and efficiency of processing large graphs is still limited. Although current systems have largely benefited from years of work in optimizations for each individual system, they still suffer from efficiency and/or scale problems. Offering superior performance when facing large-scale graph data is highly desired.
+
+## What can GraphScope Do 
+
+In GraphScope, Graph Analytics Engine (GAE) is responsible for handling graph analytics algorithms.It addresses the abovementioned challenges in the following ways:

```

Comment:
If you make them as bullet or subsection,
describe "how achieved this" in `Design of GAE` and give a link here for each subsection.